carina-api/src/main/java/com/qaprosoft/apitools/validation/XmlValidator.java

```diff
@@ -0,0 +1,35 @@
+package com.qaprosoft.apitools.validation;
+
+import org.apache.log4j.Logger;
+import org.xml.sax.SAXException;
+
+import javax.xml.XMLConstants;
+import javax.xml.transform.stream.StreamSource;
+import javax.xml.validation.Schema;
+import javax.xml.validation.SchemaFactory;
+import javax.xml.validation.Validator;
+import java.io.File;
+import java.io.IOException;
+import java.io.StringReader;
+
+
+public class XmlValidator {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://ci.qaprosoft.com/sonarqube/coding_rules#rule_key=squid%3AS1118)